src/logstatsexp.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    logmeanexp(A; dims=:)
```

Comment:
This docstring indicates that the functions works for arbitrary iterators (as `logsumexp`) whereas actually it is only defined for arrays. Can you fix the docstring or, preferably, add more general definitions?